draft-ietf-cats-framework.md

```diff
@@ -560,6 +560,16 @@ More importantly, the means for identifying a flow for the purpose of ensuring i
 
 This document does not define any mechanism for defining or enforcing service contact instance affinity.
 
+## Policy Delivery
+
+As shown in Figure 1, the control plane and management plane need to communicate some policies to the data plane. The control plane and management plane can be implemented in a control center entity.
+
+If the C-PS is logically centralized, it needs to communicate some policies to the CATS-Forwarders. In this case, the network entity containing the C-PS, such as a control center, should be able to perform management and/or control functions, including computing information awareness function, service scheduling function, etc. It collects computing and network information, and schedules service based on computing and network information. The CATS-Forwarders performs data plane functions or routing functions.
```

Comment:
Not sure what is this text is trying to say.

Reply:
Some explanations: If the C-PS is logically centralized, the C-PS can be in a controller. After the CATS decision is made,  it can communicate some policy to the CATS-Forwarders.

  If the C-PS is collocated with an Ingress CATS-Forwarder, the Ingress CATS-Forwarder would implement the C-PS functions locally.